images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -0,0 +1,29 @@
+################################################################################
+##  File:  Install-Haskell.ps1
+##  Desc:  Install Haskell for Windows
+################################################################################
+
+# Get last 3 latest versions of GHC
+[Array] $ChocoVersionsOutput = & choco search ghc --allversions --limit-output
+$RawVersionsList = $ChocoVersionsOutput | Where-Object { $_.StartsWith("ghc|") } | ForEach-Object { $_.TrimStart("ghc|") }
+$VersionsList = $RawVersionsList | ForEach-Object { [Version]::Parse($_) } | Sort-Object -Descending
```

Comment:
Why can't you add `| Get-Unique | Sort-Object -Descending | Select-Object -First 3
` and get rid of $LatestGhcVersions calculation?